server.go

```diff
@@ -565,11 +565,12 @@ func newServer(listenAddrs []net.Addr, chanDB *channeldb.DB, cc *chainControl,
 			return s.FindPeer(pub)
 		},
 		NotifyWhenOnline: s.NotifyWhenOnline,
-		ProofMatureDelta: 0,
-		TrickleDelay:     time.Millisecond * time.Duration(cfg.TrickleDelay),
-		RetransmitDelay:  time.Minute * 30,
-		DB:               chanDB,
-		AnnSigner:        s.nodeSigner,
+		ProofMatureDelta:    0,
+		TrickleDelay:        time.Millisecond * time.Duration(cfg.TrickleDelay),
+		RetransmitDelay:     time.Minute * 30,
+		RebroadcastInterval: time.Hour * 24,
```

Comment:
Any reason this is the default? It could be made less spammy IMO.

Reply:
Could reasonably be made longer, but don't think one update every 24 hrs is the biggest spam problem atm.

Reply:
Sure but given that we only consider things stale after 2 weeks it could be set a bit higher.